src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -714,10 +725,17 @@ public DatasetSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
    */
   @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
   public boolean patch(UUID id, DatasetPatchRequestModel patchRequest) {
-    String sql = "UPDATE dataset SET phs_id = COALESCE(:phs_id, phs_id) WHERE id = :id";
+    String sql =
+        "UPDATE dataset SET phs_id = COALESCE(:phs_id, phs_id), "
+            + "properties = COALESCE(:properties, properties) WHERE id = :id";
 
     MapSqlParameterSource params =
-        new MapSqlParameterSource().addValue("phs_id", patchRequest.getPhsId()).addValue("id", id);
+        new MapSqlParameterSource()
+            .addValue("phs_id", patchRequest.getPhsId())
+            .addValue(
+                "properties",
+                DaoUtils.propertiesToPGobject(objectMapper, patchRequest.getProperties()))
```

Comment:
What happens if the patch request is null? Just want to make sure it doesn't do something like have an explicit `'null'` string

Reply:
Good catch! It was setting the value to "null" so I fixed that and added some more tests!